FundacionParaguaya/MentorApp
# Fixed footer in lifemap process

In this pull request we make the continu/save button stick to the bottom of the screen even when the content is long.

**To Test**
Testing on a small device/emulator makes it easier to see the sticky button as less content is needed to fill in the page. Also you can continue a draft, but since you need to cover the whole process starting a new one is easier.
1. Go to Primary Participant. 
2. Now the Continue button should stick to the bottom of the device screen and be visible at all times. You can scroll the rest of the content as you wish. When you scroll to the bottom, everything should be visible and the Continue button should not be on top of anything.
3. Tap on Continue without filling all the required fields. The errors should appear and you should not continue to the next view.
4. Fill in the fields and tap the button. Now you should navigate to the next screen.
5. Repeat steps 2 to 4 for Family member names, genders and birthdates, family location, socio economic screens (Sierra Leone survey has one page which is long), Add Priority/Achievement, Skipped screen (skip 8-10 questions to see the effect) and Overview.
